schemas/common/descriptors/search/searchdescriptor.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/searchdescriptor",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Descriptor",
+  "meta:extensible": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor"
+  ],
+  "meta:abstract": false,
+  "type": "object",
+  "description": "A descriptor that describes search annotations for specific properties of the schemas.",
+  "definitions": {
+    "searchdescriptor": {
+      "properties": {
+        "search:indexable": {
+          "title": "is Indexable",
+          "type": "boolean",
+          "description": " When present, Boolean indicating if the property is indexable / searchable",
+          "default": false
+        },
+        "search:fetchable": {
+          "title": "is Fetchable",
+          "type": "boolean",
+          "description": "When present, Boolean indicating if the property is returned by default in search query results",
+          "default": false
+        },
+        "search:filterable": {
+          "title": "is Filterable",
+          "type": "boolean",
+          "description": "When present, Boolean indicates if this property can be used as  a filter in search queries",
+          "default": false
+        },
+        "search:stored": {
+          "title": "is stored",
+          "type": "boolean",
+          "description": "When present, Boolean indicating if the property's raw values are stored in the index",
+          "default": false
+        },
+        "search:sort": {
+          "properties": {
+            "search:sortable": {
+              "title": "isSortable",
+              "type": "boolean",
+              "description": " When present, Boolean indicating if the property is sortable ",
+              "default": false
+            },
+            "search:caseinsensitive": {
+              "title": "is Case Insensitive",
+              "type": "boolean",
+              "description": "When present, Boolean indicating if the sort on the property is case insensitive",
+              "default": false
+            }
+          }
+        },
+        "search:analyzer": {
+          "properties": {
+            "search:analyzed": {
+              "title": "isAnalyzed",
+              "type": "boolean",
+              "description": " When present, Boolean indicating if the property is analyzed ",
+              "default": false
+            },
+            "search:analyzer": {
```

Comment:
In some search cases there are different analyzers for 'search' vs 'indexing', are you distinguishing between them here and avoiding issues is we are to evolve to supporting both in future. 

Refer:
https://docs.microsoft.com/en-us/azure/search/search-analyzers
```        {
           "name":"text",
           "type":"Edm.String",
           "searchable":true,
           "indexAnalyzer":"whitespace",
           "searchAnalyzer":"simple"
        },```